### DISCUSSION TITLE
Welcome to Swap Discussions!

### DISCUSSION THREAD

**Discussion by meherett** [movnetwork/swap#6] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by meherett** [comment]
# ⇋ v0.3.0  ·  [Vapor Cryptocurrency](https://github.com/meherett/swap/releases/tag/v0.3.0) is now available 🎉

**Vapor** is a high-performance sidechain of Bytom. Vapor is a layer 2 scalability solution that uses cross-chain technology to interact with the Bytom mainchain. Compared to Ethereum, the Vapor sidechain boasts faster transactions, lower costs, and less risk of congestion.

**Comment by mellifexupwardlymobile** [comment]
Hi, is this HTLC contract compatible with ERC721 or can be easily made compatible and how? E.g. Swaping ERC721 for an ERC20 token. Thanks!
